release/ray_release/configs/global_config.py

```diff
@@ -48,6 +51,8 @@ def _init_global_config(config_file: str):
         byod_gcp_cr=config_content["byod"].get("gcp_cr"),
         state_machine_aws_bucket=config_content["state_machine"]["aws_bucket"],
         aws2gce_credentials=config_content.get("credentials", {}).get("aws2gce"),
+        pipeline_premerge=config_content.get("pipeline", {}).get("premerge", []),
+        pipeline_postmerge=config_content.get("pipeline", {}).get("postmerge", []),
```

Comment:
maybe call this `premerge_pipeline_ids`?

and maybe check that a pipeline is uniquely either `premerge` or `postmerge`?